src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,61 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+    private static final int MAX_SIZE = 12;
+    private static final int EMPTY = -1;
+    private static final int FULL = 11;
+
+
+    private final int[] numbers = new int[MAX_SIZE];
+
+    public static int getMaxSize() {
+        return MAX_SIZE;
+    }
+
+    public static int getEMPTY() {
+        return EMPTY;
+    }
+
+    public static int getFULL() {
+        return FULL;
+    }
+
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = EMPTY;
```

Comment:
Errors in collaborators work (lack of good review). 
This constant should be different (separate) than the one in peekaboo and countOut methods.